### DISCUSSION TITLE
general feedback: is this really a Notion alternative?

### DISCUSSION THREAD

**Discussion by FarisHijazi** [AppFlowy-IO/AppFlowy#1271] [General]
Hi, I totally support this effort of having an open source Notion alternative (for many reasons).

Now I'm noticing that AppFlowy isn't a Notion copy, Notion is a markdown based text editor with extra features like databases and nested pages.

Now for AppFlowy to actually be a replacement for notion and to add value, it's missing a few things:
- it needs to be a markdown editor (`-` would start a bullet list, "```" would start a code block, `#` would start a heading 1, `/` would open the menu of all types of blocks), we shouldn't need to use those buttons at the bottom, they should be only when getting started for noobs, we should have the option to use either one, buttons or markdown shortcuts.
- the notion block system is great, you can drag them and replace them, it's clear which lines are together and which lines are in different paragraphs
- DATABASES!! I can't stress this enough, I would use any other markdown editor if it wasn't for the databases in Notion and the different views it can make
- (lower priority in the short term): mobile app, hopefully this won't be too hard since it's written in flutter

Before you think of addressing these issues I mentioned, the authors should all get together and really decide, what is the vision and end goal of AppFlowy? is it to be a Notion alternative? or just a nice text editor?

Because this will dictate its success, you need to know you target audience, and since you're marketing it in the frontpage as a Notion-alternative, this is what users are going to expect

**Comment by annieappflowy** [comment]
Hi @FarisHijazi , first of all, thanks a lot for your feedback. We're working toward offering a more complete set of Notion-like features, but it takes time. :) There are a few things on your list we are developing:

1. Markdown is already supported in [AppFlowy Editor](https://pub.dev/packages/appflowy_editor), a standalone rich-text editor we publish for Flutter developers. We're integrating it into AppFlowy to replace flutter_quill, the editor with the toolbar at the bottom you are using in the latest release. After the integration, you will be able to use markdowns with "/" command. See this [pub dev page](https://pub.dev/packages/appflowy_editor) for more visual details
2. Databases: we currently support table-view and kanban-view. 
![grid-database](https://user-images.githubusercontent.com/12026239/195484150-38837994-b86a-4c34-8d8e-99f855856fae.png)
![v0 0 5_appflowy_kanban_board](https://user-images.githubusercontent.com/12026239/195484121-7f93d8fa-59be-4aa2-a664-e1585f515cd9.gif)
3. The mobile app is under design. It should come out pretty soon, as you said AppFlowy is built with Flutter.

The goal for us is to offer an open-source Notion alternative with an emphasis on data ownership and customizations.  I'm happy to exchange ideas with you and answer any questions you may have about AppFlowy.

**Reply by FarisHijazi** [reply]
Thanks for replying, i totally understand that things take time, I'm just trying to provide rich feedback.

Well that sounds great that things are moving smolthely

Two last thing to mention, is that i believe each new page should be a page, rather than choosing between a grid and a page.

And the other is templates, i know some people that prefer notion over other tools solely because of the templates feature.
But this can be added later, the markdown editing is the most crucial.

And of course, thank you for bringing a toool like this to the community

**Reply by annieappflowy** [reply]
Hi @FarisHijazi , [v0.0.6](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.0.6) is out. This release includes a major upgrade to AppFlowy's editing experience. It now supports basic Markdown syntax. Let us know your feedback after giving it a try!

Integrating Kanban Board and Grid into pages is scheduled to deliver at v0.0.9

And templates will be a mentorship program project for the community. 

Very much appreciate your rich feedback. Look forward to seeing you often in the community :D

**Reply by FarisHijazi** [reply]
Thanks! Will get on it when I get back

**Comment by ARGF0RCE** [comment]
Even I agree with you @FarisHijazi. That is why I opened this [FR issue](https://github.com/AppFlowy-IO/AppFlowy/issues/1332) and am determined to resolve this by working together with the community. So, I would be glad if we can form a team inclusive of yourself in working at the addition of this feature which will be a sure awesome perk for the app. I am sure forming a team would be beneficial with at least one of the collaborators on board. What do you say, @FarisHijazi @annieappflowy and the community?!

**Reply by FarisHijazi** [reply]
I'm happy to test and debug and research, but I'm not a flutter dev

**Reply by ARGF0RCE** [reply]
Awesome that would also be a benefit regardless.

**Reply by annieappflowy** [reply]
Getting back to you on [your FR issue](https://github.com/AppFlowy-IO/AppFlowy/issues/1332)

**Comment by annieappflowy** [comment]
@ARGF0RCE , participation from the community is always welcome! You can check out a list of newcomer friendly issues [here ](https://github.com/orgs/AppFlowy-IO/projects/5/views/4). If you can't find an interesting issue to tackle, you can create a feature request to scope your work, and we will provide help and feedback.

**Comment by Lutif** [comment]
is drag and drop of blocks implemented ? or on road map ?

**Reply by annieappflowy** [reply]
yes, it's included in our 2024 plan

**Comment by peanutyost** [comment]
Is there a way to add note from a api call or from n8n.io?
